kaggle-classification/keras_trainer/model.py

```diff
@@ -128,7 +129,7 @@ def _prep_texts(self, texts):
   def _load_model(self):
     try:
       tf.gfile.Copy(self.model_path, TEMPORARY_MODEL_PATH, overwrite=True)
-      self.model = load_model(TEMPORARY_MODEL_PATH)
+      self.model = load_model(TEMPORARY_MODEL_PATH, custom_objects={'auc_roc': auc_roc})
```

Comment:
Why do you have to pass this in here?

Reply:
The loading fails otherwise with: "ValueError: Unknown metric function:auc_roc".

All custom objects part of the model need to be passed in here.